write-fonts/src/tables/variations/ivs_builder.rs

```diff
@@ -329,6 +329,15 @@ impl<'a> Encoder<'a> {
             log::trace!("{:?}", DebugTodoList(&to_process));
         }
         self.encodings = to_process.into_iter().flatten().collect();
+        // now sort the items in each individual encoding
+        self.encodings
+            .iter_mut()
+            .for_each(|enc| enc.deltas.sort_unstable());
+        // and then sort the encodings themselves; order doesn't matter,
+        // but we want to match fonttools output when comparing ttx
+        self.encodings.sort_unstable_by(|a, b| {
+            (a.shape.row_cost(), &a.shape).cmp(&(b.shape.row_cost(), &b.shape))
```

Comment:
I think this line won't match fonttools. As we discussed, comparing `shape` here compares lower indices first. Whereas in fonttools the `bits` encoding compares higher indices first, because it's compared as a big integer. No?